src/stepFunctions/commands/visualizeStateMachine/getStateMachineDefinitionFromCfnTemplate.ts

```diff
@@ -0,0 +1,72 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as fs from 'fs'
+import { getLogger, Logger } from '../../../shared/logger'
+
+/**
+ * @param {string} uniqueIdentifier - unique identifier of state machine
+ * @param {string} templatePath - path for the template.json file
+ *
+ * @returns the escaped ASL Json definition string of the state machine construct
+ */
+export function getStateMachineDefinitionFromCfnTemplate(
+    uniqueIdentifier: string,
+    templatePath: string
+): string | undefined {
```

Comment:
This return type forces you to have a lot of workarounds for returning `undefined` (`as string`, ternary statements to return a blank string). I'm thinking you may want to retype this to just return `string` and either have this function throw the error or return a `''` from the catch block.